lib/install.js

```diff
@@ -464,6 +448,15 @@ const fs = require('fs'),
         }
     };
 
+    function makeSession(options, next){
+        log.info("install#makeSession");
+        if (!options.session) {
+            options.session = new novacom.Session(options.device, next);
+        } else {
+            next(null, options.session);
```

Comment:
Need to check ares-install opkg command.
Cause 375 line is "function(next)"

Reply:
I've checked opkg insttall and that is working fine.